tools/backups2datalad/util.py

```diff
@@ -397,3 +422,126 @@ def exp_wait(
     while attempts is None or n < attempts:
         yield base ** n * multiplier
         n += 1
+
+
+async def arequest(client: httpx.AsyncClient, method: str, url: str) -> httpx.Response:
+    waits = exp_wait(attempts=5, base=1.25, multiplier=1.25)
+    while True:
+        try:
+            r = await client.request(method, url, follow_redirects=True)
+            r.raise_for_status()
+        except httpx.HTTPError as e:
+            if isinstance(e, httpx.RequestError) or (
+                isinstance(e, httpx.HTTPStatusError) and e.response.status_code >= 500
+            ):
+                try:
+                    delay = next(waits)
+                except StopIteration:
+                    raise e
+                log.warning(
+                    "Retrying %s request to %s in %f seconds as it raised %s: %s",
+                    method.upper(),
+                    url,
+                    delay,
+                    type(e).__name__,
+                    str(e),
+                )
+                await trio.sleep(delay)
+                continue
+            else:
+                raise
+        return r
+
+
+@dataclass
+class Remote:
+    name: str
```

Comment:
FTR I thought to suggest: if this is the one to be used only for backup special remotes, then let's make it `RcloneRemote` and add explicit `rclone_name` since we will be reusing the same rclone target (configured in `~/.rclonerc` of the user), but then decided to just "workaround" for now/ever and have that rclone config for `dandi-dandisets-dropbox` duplicated for `dandi-dandizarrs-dropbox` and `rclone ls dandi-dandizarrs-dropbox:` seems to work just as fine, so let's keep as is (in this PR) and not bring separate names here.